src/main/plugins/org.dita.pdf2/xsl/fo/troubleshooting-elements.xsl

```diff
@@ -0,0 +1,205 @@
+            <fo:block xsl:use-attribute-sets="condition__content">
+              <xsl:apply-templates/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>
+
+    <xsl:template match="*[contains(@class, ' troubleshooting/cause ')]">
+        <fo:block xsl:use-attribute-sets="cause">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:apply-templates select="." mode="dita2xslfo:task-heading">
+                <xsl:with-param name="use-label">
+                    <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                        <xsl:with-param name="pdf2-string">Troubleshooting Cause</xsl:with-param>
+                        <xsl:with-param name="common-string">trouble_cause</xsl:with-param>
+                    </xsl:apply-templates>
+                </xsl:with-param>
+            </xsl:apply-templates>
+            <fo:block xsl:use-attribute-sets="cause__content">
+              <xsl:apply-templates/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>   
+    
+    <xsl:template match="*[contains(@class, ' troubleshooting/remedy ')]">
+        <fo:block xsl:use-attribute-sets="remedy">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:apply-templates select="." mode="dita2xslfo:task-heading">
+                <xsl:with-param name="use-label">
+                    <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                        <xsl:with-param name="pdf2-string">Troubleshooting Remedy</xsl:with-param>
+                        <xsl:with-param name="common-string">trouble_remedy</xsl:with-param>
+                    </xsl:apply-templates>
+                </xsl:with-param>
+            </xsl:apply-templates>
+            <fo:block xsl:use-attribute-sets="remedy__content">
+                <xsl:apply-templates/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>
+    
+    <xsl:template match="*[contains(@class, ' troubleshooting/responsibleParty ')]">
+        <fo:block xsl:use-attribute-sets="responsibleParty">
+            <xsl:call-template name="commonattributes"/>
+            <xsl:apply-templates select="." mode="dita2xslfo:task-heading">
+                <xsl:with-param name="use-label">
+                    <xsl:apply-templates select="." mode="dita2xslfo:retrieve-task-heading">
+                        <xsl:with-param name="pdf2-string">Troubleshooting responsibleParty</xsl:with-param>
+                        <xsl:with-param name="common-string">trouble_responsibleParty</xsl:with-param>
+                    </xsl:apply-templates>
+                </xsl:with-param>
+            </xsl:apply-templates>
+            <fo:block xsl:use-attribute-sets="responsibleParty__content">
+                <xsl:apply-templates/>
+            </fo:block>
+        </fo:block>
+    </xsl:template>
+
+    <xsl:template match="*[contains(@class, ' task/steptroubleshooting ')]">

```

Comment:
A rule for task elements should not be part of the troubleshooting override, unless it's made specific to the `troubleshooting` topic context. This looks like it should be in the task override.